blcheck

```diff
@@ -378,150 +400,192 @@
         bsb.empty.us
         88.blocklist.zap"
 
+# reputation/scoring pages
+# score.senderscore.com
 
-        # reputation/scoring pages
-        # score.senderscore.com
-
-    CONF_BLACKLISTS_TEST="
-        rbl.rbldns.ru
-        rbl.rbldns.ru=127\.0\.0\.1
-        rbl.rbldns.ru#DNSWL
-        black.list.com=127.0.[0-9]+.0#URIBL
-        dbl.spamhaus.org#URIBL
-        t3direct.dnsbl.net.au=127\.0\.0\.1#DNSWL
-        t3direct.dnsbl.net.au#BLA
-        t3direct.dnsbl.net.au%127\.0\.0\.1#DNSWL"
-
+#CONF_BLACKLISTS_TEST="
+#        rbl.rbldns.ru
+#        rbl.rbldns.ru=127\.0\.0\.1
+#        rbl.rbldns.ru#DNSWL
+#        black.list.com=127.0.[0-9]+.0#URIBL
+#        dbl.spamhaus.org#URIBL
+#        t3direct.dnsbl.net.au=127\.0\.0\.1#DNSWL
+#        t3direct.dnsbl.net.au#BLA
+#        t3direct.dnsbl.net.au%127\.0\.0\.1#DNSWL"
 
 #~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ }
 
 
 # Definitions {
 
-    # Common regular expressions
-    REGEX_IP='\([0-9]\{1,3\}\)\.\([0-9]\{1,3\}\)\.\([0-9]\{1,3\}\)\.\([0-9]\{1,3\}\)'
-    REGEX_DOMAIN='\([a-zA-Z0-9]\+\(-[a-zA-Z0-9]\+\)*\.\)\+[a-zA-Z]\{2,\}'
-    REGEX_TDL='\([a-zA-Z0-9]\+\(-[a-zA-Z0-9]\+\)*\.\)[a-zA-Z]\{2,\}$'
-    REGEX_LIST='^[ \\t]*([^=#]+)(=(([^#])+))?(#(DNSBL|DNSWL|URIBL))?[ \\t]*$'
-
-    # Colors
-    if [[ $- == *i* ]]; then
-        RED=$(tput setaf 1)
-        GREEN=$(tput setaf 2)
-        YELLOW=$(tput setaf 3)
-        CLEAR=$(tput sgr0)
-    else
-        RED=$(tput -T xterm setaf 1)
-        GREEN=$(tput -T xterm setaf 2)
-        YELLOW=$(tput -T xterm setaf 3)
-        CLEAR=$(tput -T xterm sgr0)
-    fi
+# Common regular expressions
+REGEX_IP='\([0-9]\{1,3\}\)\.\([0-9]\{1,3\}\)\.\([0-9]\{1,3\}\)\.\([0-9]\{1,3\}\)'
+REGEX_IPV6='\([0-9a-fA-F]\{0,4\}:\)\{1,7\}[0-9a-fA-F]\{0,4\}'
```

Comment:
Note this regex is only used to filter output. IPv6 regex is not this simple because it also has compressed and uncompressed regex. Making this regex too complex has no benefit at this point.